docs/samples/config/msidp/msidp_settings.py.sample

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""
+
+    mslib.msidp.conf.py.example
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    config for msidp
+
+    This file is part of mss.
+
+    :copyright: Copyright 2019 Shivashis Padhi
+    :copyright: Copyright 2019-2024 by the MSS team, see AUTHORS.
```

Comment:
Author and year should be updated.

Reply:
Don't understand the comment with author and year. 

Can you give an example?

Reply:
comment removed, correct, I was blind